data/hails.txt

```diff
@@ -1009,7 +1009,9 @@ phrase "friendly civilian"
 		"Some crazy militia officer "
 		"Walking into the spaceport, someone "
 	word
-		"asked me if I would help her fight local pirates. That's too dangerous for me!"
+		"asked me if I would help them fight local pirates. That's too dangerous for me!"
```

Comment:
if you leave them out the sentence makes sence
"aske me if I would help fight local pirates"